### ISSUE TITLE
darkplaces-text-0.2 test suite failure

### ISSUE THREAD

**Issue by peti** [bacher09/darkplaces-text#1]
See http://hydra.cryp.to/build/887218/log/raw for a complete build log that shows the error.

**Comment by bacher09** [comment]
Thanks for bug report.
More simplified test looks like this:

```
>  toText $ DPText [HexColor 41,DPString "F"]
"^x29F"
> parseDPText "^x29F"
DPText [HexColor 671]
```

Darkplaces requires hex token to be exactly 3 digits, like `^x029`.
It also seems that the are another bug in tests, more precisely in QuickTest `Arbitrary` instance,
since darkplaces do not support any kind of escaping tokens.

**Comment by bacher09** [comment]
Issue is fixed in 0.2.1
